module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -124,6 +124,13 @@ abstract class Options implements TranslatorAwareInterface
      */
     protected $retainFiltersByDefault = true;
 
+    /**
+     * Should we display a "Reset Filters" link regardless of retainFiltersByDefault?
+     *
+     * @var bool
+     */
+    protected $alwaysDisplayResetFilters = false;
```

Comment:
Since this is initialized by the constructor, it doesn't really need a default value.

Reply:
Fixed.